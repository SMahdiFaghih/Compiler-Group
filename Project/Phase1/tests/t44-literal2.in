01010101
0xabcdef
x0xabcde
3.141592
6.2831E2
9.424E-2
"breath"
truefalse
falsetrue
