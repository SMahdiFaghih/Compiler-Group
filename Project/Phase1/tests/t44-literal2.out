T_INTLITERAL 01010101
T_INTLITERAL 0xabcdef
T_ID x0xabcde
T_DOUBLELITERAL 3.141592
T_DOUBLELITERAL 6.2831E2
T_DOUBLELITERAL 9.424E-2
T_STRINGLITERAL "breath"
T_ID truefalse
T_ID falsetrue
